test.tf

```diff
@@ -29,10 +29,11 @@ resource "aws_network_interface" "foo" {
 
 resource "aws_ami" "awsAmiEncrypted" {
```

Comment:
**Tenable.cs remediation**
 
**[MEDIUM]: Ensure encryption is enabled for AWS Amazon Machine Image (AMI)**
 
**Vulnerability**
AWS AMI does not have encryption enabled. This may lead to data leak.
 
**Remediation**
AWS Amazon Machine Images (AMI) should be encrypted using encrypted EBS snapshots. AMI's created from encrypted EBS snapshots are encrypted by default. Ensure using AWS KMS for encryption of the EBS snapshots to create encrypted AMI.
 In Terraform - set 'ebs_block_device.encrypted' to 'true'.

source: tenablecs-